cmake/LLVMIRUtil.cmake

```diff
@@ -111,6 +111,33 @@ function(llvmir_attach_bc_target)
   # compile lang flags
   llvmir_extract_lang_flags(IN_LANG_FLAGS ${LINKER_LANGUAGE})
 
+  # Link static libraries
+  if(LLVMIR_ATTACH_LINK_STATIC_LIB)
+    message(STATUS "LINK_STATIC_LIB option is turned on. Linking static libraries.")
```

Comment:
Adjust this message appropriately, i.e.,

`X option is turned ON. Attaching to dependent static libraries of target Y`

Please fill in `X` and `Y`.

Reply:
Modified as suggested in the new commit